dpnp/dparray.pxd

```diff
@@ -32,7 +32,7 @@ Represents multi-dimensional array using USM interface for an Intel GPU device.
 
 
 from libcpp.vector cimport vector
-
+from dpctl cimport SyclQueue
```

Comment:
What happened if no `dpctl` exists?

Reply:
This change makes `dpctl` a hard build-time dependency of `dpnp`.